opcplc/nodesfile.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$type": "OpcPlc.ConfigFolder, opcplc",
+  "NamespaceId": 7,
+  "Name": "MyTelemetry",
```

Comment:
this is not picked up and created  at least when i was running. either you debug and fix it or remove it.